### DISCUSSION TITLE
FDNS + openSUSE Tumbleweed

### DISCUSSION THREAD

**Discussion by kzsa** [netblue30/firejail#6240] [Q&A]
Hey all,
I need help installing and using FDNS + openSUSE Tumbleweed, because both suse and fdns are new to me.
Is there anything I need to be aware of regarding the said distro?
(I've used Debian based systems so far.)
Maybe something specific about compiling from source?
Any information would be greatly appreciated,
Thanks Zsolt

**Comment by ghost** [comment]
I've only briefly touched OpenSUSE Tumbleweed a while back but there's a few online resources you might try:

- [find packages](https://software.opensuse.org/package/fdns) shows only one fdns package, somewhat old but it has a '1 Click-Install' button
- if you want to build from source, see the [fdns wiki](https://github.com/netblue30/fdns/wiki/Download-and-Install)

You might get better help over at the [fdns](https://github.com/netblue30/fdns) repo/issue tracker.

HTH

**Reply by kzsa** [reply]
Thank you very much, I will compile it and see what happens. :D
Br: Zsolt
